README.rst

```diff
@@ -14,18 +14,78 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.2.6 or newer.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
```

Comment:
> installs into the OMERO.web framework" sounds like omero_figure is being placed into an existing OMERO.web directory. But when you do pip install omero_figure this is completely independent of OMERO.web

@will-moore I would like to kindly remind you that we are talking about python packages. It is not about _placing/coping_ folders but PYTHONPATH. I think install is the right word as the process adds omero-figure to PYTHONPATH and update OMERO.web config that OMERO.web consumes on startup.